src/cache.service.ts

```diff
@@ -1,23 +1,31 @@
 import { Injectable } from '@angular/core';
 import { SqlStorage } from './storage';
-import { Observable, Subject } from 'rxjs/Rx';
+import { Observable } from 'rxjs/Observable';
+import { Subject } from 'rxjs/Subject';
 import { Request, Response, ResponseOptions } from '@angular/http';
-
-const MESSAGES = {
+import 'rxjs/add/observable/fromPromise';
```

Comment:
Is this imports really necessary? I think this operators is already hidden under rxjs/Observable